ring_buffer/ring_buffer.py

```diff
@@ -7,8 +7,17 @@ def __init__(self, capacity):
         self.current = None
         self.storage = DoublyLinkedList()
 
-    def append(self, item):
-        pass
+    def append(self, item):  # need to append before we get
+        # if ring buffer is empty, and current isnt set, set current to be head
```

Comment:
Let's see if this pseudo code helps:
First we need to check to see if we have anything in our storage. 
If our storage is empty, we need to set the item we're appending to the head.

Next we could check to see if the storage is at capacity. 

If it's not add an item to the tail of the list.

If storage is at capacity, we need to check if there is a next node (because of our pointer). 
Remember... we don't have to replace the ENTIRE node and change/make new pointers. We could just update the value of the REFERENCE to the object in self.current, which is basically a pointer for our LRU (least recently used) item.